Makefile

```diff
@@ -30,4 +30,19 @@ resource_files:
 		file=$${f%.yaml}; \
 		hugo new -k resource content/resources/$${file}.md ; \
 	done
-	hugo new -k all_the_resources content/resources/_index.md ;
\ No newline at end of file
+	hugo new -k all_the_resources content/resources/_index.md ;
+
+# Generates blank Cookstyle Cop pages, one for each yaml file in
+# _vendor/github.com/chef/cookstyle/docs-chef-io/data/cookstyle directory
+# And an index page that lists all of the cops.
+cookstyle_cops_pages:
+	rm -rf content/workstation/cookstyle/*
+	dataDir=_vendor/github.com/chef/cookstyle/docs-chef-io/data/cookstyle; \
```

Comment:
I'm getting two entries for Cookstyle in the left nav. 
<img width="196" alt="Screen Shot 2021-02-04 at 11 37 13 AM" src="https://user-images.githubusercontent.com/4400151/106949244-90538700-66e1-11eb-89b3-6506d5a98183.png">

Reply:
See chef/chef-workstation#1678